config.py

```diff
@@ -61,7 +61,11 @@
 
 # This generates the name for the raw files
 # Note that there is no underscore between study_name and run.
+
+# XXX this can likely be done more efficiently
 base_raw_fname = '{subject}_' + study_name + '{run}_raw.fif'
+base_epochs_fname = '{subject}_' + study_name + '_epo.fif'
+base_ave_fname = '{subject}_' + study_name + '_ave.fif'
```

Comment:
convention is to name it -epo.fif and -ave.fif (not with _)

Reply:
Ok. We should find a more consistent/ efficient way for the filenames anyways (see issue #16). 
Ideally, we just have one base-name and add the endings/ extensions in every script.